_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,308 @@ nav_order: 50
 
 ## Getting started 
 
-Before getting started, you must have:
+Before working with alerting dashboards and visualizations, ensure that you have:
 
 - Installed OpenSearch and OpenSearch Dashboards version 2.9 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
-- Installed the Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+- Installed the Alerting, Anomaly Detection, and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/).
+- Started your local environment. The [OpenSearch Playground](https://playground.opensearch.org/app/home) is read-only, so you should use your local environment to perform the steps in the following tutorials.
 
 ## Configuring admin settings
 
-Users can only access, create, or manage alerts for resources for which they have permissions. Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards permissions. It is enabled by default and appears as a feature under **Dashboards Management** > **Advanced Settings** > **Visualization**. If the setting is disabled, it does not appear. You can disable the setting at the cluster level in the `opensearch-dashboards.yml` file.
+Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards permissions. You can only access, create, or manage alerts on resources for which you have appropriate permissions.
+
+The alerting dashboards and visualizations setting is enabled by default and can be configured from **Dashboards Management** > **Advanced Settings** > **Visualization**. 
+
+To disable the setting at the cluster level, set `vis_augmenter.pluginAugmentationEnabled: false` in your `opensearch-dashboards.yml` file. If the setting is disabled, the feature does not appear in the settings pane. 
 
 ## General requirements for alerting visualizations
 
-Alerting visualizations are displayed as time-series charts that give you a snapshot of the alert, alert status, last updated time, and reason for the alert. You can display up to 10 metrics on your chart, and each series can be shown as a line on the chart.
+Alerting visualizations are displayed as time-series charts that provide a visual representation of alerts, their status, last updated time, and the reason for the alert.
+
+Consider the following general requirements for setting up or creating alerting visualizations: 
+
+- The visualization must use the [Vizlib Line Chart](https://documentation.insightsoftware.com/vizlibdocumentation/Content/Library/Line_Chart/Vizlib_Line_Chart_-_Overview.htm) extension.
```

Comment:
Should we add a little more information about this requirement? Is there any special setup needed on the client side?

Reply:
@AWSHurneyt Please provide feedback on the above question.

Reply:
@lezzago this question seems related to the feature anywhere visualizations. Could you comment?

Reply:
@vagimeli these are the normal line charts. Vizlib is mentioned because it is the visualization library behind rendering them. There should be no need to add more information about it. We can even remove the `Vizlib` name since it might add more confusion to the user.